src/components/contact_page/ContactPage.jsx

```diff
@@ -3,11 +3,57 @@ import './ContactPage.css';
 import Navigator from '../general/Navigator';
 import Footer from '../general/Footer';
 import Form from './Form';
+import { createStore } from "redux";
+import { Provider } from "react-redux";
+
+
+
+const initialState = {
+    name: '',
+    email: '',
+    subject: '',
+    message: '',
+}
+
+function reducer(state = initialState, action) {
+switch(action.type) {
```

Comment:
It's not really good to use Redux for just a form. But lets say it's okay in your case, cause you app doesn't really need it.